include/star/single_star.h

```diff
@@ -493,11 +493,16 @@ void extract_line_text(stellar_type&, real& z, real&, real&, real&,
 bool merge_elements(single_star* primary, single_star *secondary);
 
 void  addstar(node*,
-	      real t_rel=0,
+	      real t_cur=0,
```

Comment:
Renamed `t_rel` to `t_cur` to more closely match the function's name in add_star.C and to prevent clashing with the "actual" `t_rel`.